### ISSUE TITLE
Auto-save loading bug after evolving

### ISSUE THREAD

**Issue by Teashrock** [Revolutionary-Games/Thrive#3729]
**Describe the bug**

Tried to load an auto-saved game, got an error.

**To Reproduce**
Steps to reproduce the behaviour:
1. Start new game.
2. Play to a certain moment, where you reproduced and evolved,
3. Try to load the autosave of right after your evolved.
4. See the error.

**Expected behaviour**
1. Start new game.
2. Play to a certain moment, where you reproduced and evolved,
3. Try to load the autosave of right after your evolved.
4. The save loads as intended and you continue playing.

**System info (please complete the following information):**
- OS:  Windows 10 x64
- Graphics card: 
    OpenGL ES 3.0 Renderer: AMD Radeon (TM) R5 M330
- Thrive version: 0.5.9.0

**Logs**

```
Помилка завантаження - З'явилося заперечення, під час обробки завантажених об'єктів exception: InstanceNotLoadedYetException: Operation is not valid due to the current state of the object.
  at FloatingChunk.get_EntityGraphics () [0x0000f] in <5208d8567ee64dd29346dd4ccb964802>:0 
  at Microbe._Ready () [0x00310] in <5208d8567ee64dd29346dd4ccb964802>:0 
  at (wrapper managed-to-native) Godot.NativeCalls.godot_icall_2_441(intptr,intptr,intptr,bool)
  at Godot.Node.AddChild (Godot.Node node, System.Boolean legibleUniqueName) [0x00011] in <255400c4320c44b8897009d693c12504>:0 
  at SceneManager.SwitchToScene (Godot.Node newSceneRoot, System.Boolean keepOldRoot) [0x00027] in <5208d8567ee64dd29346dd4ccb964802>:0 
  at InProgressLoad.Step () [0x001ac] in <5208d8567ee64dd29346dd4ccb964802>:0 
```
Full game log is here: https://pastebin.com/QdbWsqeP

**Comment by hhyyrylainen** [comment]
This looks like the engulfment loading problems, which should be already fixed, right @Kasterisk ?

**Comment by Kasterisk** [comment]
Yeah, I believe this should be fixed with #3598. Bug reproduction steps and error logs obviously look similar.
